pkg/tftarget/tftarget.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // Name is the target name for TFTarget
-const Name = "validation.terraform.forsetisecurity.org"
+const Name = "validation.resourcechange.terraform.google.com"
```

Comment:
```suggestion
const Name = "validation.resourcechange.terraform.cloud.google.com"
```

Just in case Google launches some "terraform" product unrelated to cloud.

Reply:
works for me.